app/scripts.babel/bkg/common.js

```diff
@@ -16,13 +16,40 @@ limitations under the License.
 
 const nxPattern = /(^https?:\/\/[A-Za-z_\.0-9:-]+\/[A-Za-z_\.0-9-]+\/)(?:(?:nxdoc|nxpath|nxsearch|nxadmin|nxhome|nxdam|nxdamid|site\/[A-Za-z_\.0-9-]+)\/[A-Za-z_\.0-9-]+|view_documents\.faces|ui\/#!\/|[a-zA-Z0-9_%]+\/?|view_domains\.faces|home\.html|view_home\.faces)/;
 
-const window = window || {}; // eslint-disable-line no-use-before-define
 window.app = window.app || {};
 const studioExt = window.studioExt = window.studioExt || {};
 
 let dependencies;
 
-const notification = window.notification = (idP, titleP, messageP, img, interaction) => {
+let CONNECT_DOMAIN = 'connect.nuxeo.com';
+let CONNECT_URL = `https://${CONNECT_DOMAIN}`;
+
+chrome.storage.sync.get('value', (res) => {
+  if (res.value && res.value.length > 0) {
+    CONNECT_DOMAIN = res.value;
+    CONNECT_URL = `https://${CONNECT_DOMAIN}`;
+    return
+  } else {
+    CONNECT_DOMAIN = 'connect.nuxeo.com';
+    CONNECT_URL = `https://${CONNECT_DOMAIN}`;
+  }
+});
+
+const getStudioUrl = window.setStudioUrl = () => {
```

Comment:
This is not used, and also, it seems it wouldn't work as is.

The return is happening in a callback, so it wouldn't be the function's return (AFAICT).

For this method and the next (`setStudioUrl`) it might be required to accept a callback to be run with the result as parameter, since these are making use of asynchronous code.